alchemy-db-mongo/src/test/java/io/rtr/alchemy/db/mongo/RevisionManagerTest.java

```diff
@@ -66,7 +65,7 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
                 revision = (Long) entity.value;
                 return null;
             }
-        }).when(ds).insert(any());
+        }).when(ds).insert(any(Object.class));
```

Comment:
Is this just to prevent nulls from matching?

Reply:
The method was overloaded in a later version of the MongoDB driver, so, the method became ambiguous.